molecule/default/prepare.yml

```diff
@@ -3,5 +3,8 @@
   
   tasks:
     - package:
-        name: ca-certificates
-        state: latest
\ No newline at end of file
+        name: {{ item }}
+        state: latest
+        loop:
+        - ca-certificates
+        - systemd
```

Comment:
I'm not real familiar with Ubuntu but doesn't Ubuntu tend to bump packages willy-nilly during the life cycle of something like 20.04?  I ask because I worry about forcing systemd to latest if it might receive some unexpected update.  Looks like might be gap in deb packages, as RPMs require systemd but I don't see deb package require systemd.

Reply:
This is just a part of the molecule tests. So it's a prepare on those test containers.

Previous tests failed with apache2 in a weird state that seems to have something to do with systemd.

Reply:
Trying to fix this

https://github.com/OSC/ood-ansible/runs/7472899360?check_suite_focus=true#step:6:396

Reply:
Is there a way to get something like `journalctl -u apache2` or just `systemctl status apache`?  I know one thing that bit me with Apache2 on the Puppet module is have to ensure either leave defaults for log path in ood-portal-generator or ensure the path for Ubuntu is correct (the Puppet module always defines a path so had to be more dynamic).

Reply:
Ya know I think it actually does start correctly there's just something in the way ansible is verifying. If you google `ansible apache2 Service is in unknown state` you'll get lots of hits with systemd + ansible + containers.

Worst case I put a flag here where we can ignore the failure on test, but I'm going to try podman and/or find ansible provisioner flags.

Reply:
just using `podman` instead of `docker` fixed this